src/librustc_target/abi/call/mod.rs

```diff
@@ -44,6 +44,8 @@ pub enum PassMode {
     Cast(CastTarget),
     /// Pass the argument indirectly via a hidden pointer.
     Indirect(ArgAttributes),
+    /// Pass the unsized argument indirectly via a hidden pointer.
+    UnsizedIndirect(ArgAttributes, ArgAttributes),
```

Comment:
The second `ArgAttributes` should be an `Option<ArgAttributes>` inside `Indirect`.